README.md

```diff
@@ -103,12 +103,8 @@ It is recommended to leave this optional configuration set to `false` unless abs
 Setting `ssl_verify_none` to `true` causes the output plugin to skip certificate verification when sending log ingest requests to SSL and TLS protected HTTPS endpoints.
 This option may be required if you are using a self-signed certificate, an expired certificate, or a certificate which was generated for a different domain than the one in use.
 
-### `codec` 
```

Comment:
We never used this option. Not sure why it was documented in the first place. I think it maybe came from the output plugin template.